src/main/resources/log4j2.fabric.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="WARN" packages="com.mojang.util,net.minecrell.terminalconsole.util">
+	<Appenders>
+
+		<!--	System out	-->
+		<Console name="SysOut" target="SYSTEM_OUT">
```

Comment:
Will we ship a separate log4j2 config for vanilla launcher, or is it the launcher's responsibility to specify log4j config?

Reply:
I think that might be the best solution. For now im concentrating on dev time.